frontend/src/app/components/work-packages/wp-relations-count/wp-relations-count.component.ts

```diff
@@ -1,20 +1,20 @@
-import {Component, Input, OnDestroy, OnInit} from '@angular/core';
-import {componentDestroyed} from 'ng2-rx-componentdestroyed';
-import {takeUntil} from 'rxjs/operators';
+import {Component, Input, OnInit} from '@angular/core';
 import {WorkPackageRelationsService} from '../../wp-relations/wp-relations.service';
 import {WorkPackageCacheService} from '../../work-packages/work-package-cache.service';
 import {combineLatest} from 'rxjs';
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/components/work-packages/wp-relations-count/wp-relations-count.component.ts#L6"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>